src/external/dftk_to_wannier90.jl

```diff
@@ -0,0 +1,605 @@
+    push!(angular_parts, [2,3] => (θ,φ) -> √(15/(4π))*sin(θ)*cos(θ)*sin(φ) )   #dyz
+    push!(angular_parts, [2,4] => dx2_y2 )                                     #dx2-y2
+    push!(angular_parts, [2,5] => (θ,φ) -> √(15/(16π))*(sin(θ)^2)*sin(2*φ) )   #dxy
+
+    push!(angular_parts, [3,1] => (θ,φ) -> (√7/(4*√(2π)))  * (5*cos(θ)^3 - 3*cos(θ)) )                   #fz3  
+    push!(angular_parts, [3,2] => (θ,φ) -> (√21/(4*√(2π))) * (5*cos(θ)^2 - 1)*sin(θ)*cos(φ) )            #fxz2
+    push!(angular_parts, [3,3] => (θ,φ) -> (√21/(4*√(2π))) * (5*cos(θ)^2 - 1)*sin(θ)*sin(φ) )            #fyz2
+    push!(angular_parts, [3,4] => (θ,φ) -> (√105/(4*√π))   * (sin(θ)^2)*cos(θ)*cos(2*φ) )                #fz(x2-y2)
+    push!(angular_parts, [3,5] => (θ,φ) -> (√105/(4*√π))   * (sin(θ)^2)*cos(θ)*sin(2*φ) )                #fxyz
+    push!(angular_parts, [3,6] => (θ,φ) -> (√35/(4*√(2π))) * (sin(θ)^3)*(cos(φ)^2 - 3*sin(φ)^2)*cos(φ) ) #fx(x2-3y2)
+    push!(angular_parts, [3,7] => (θ,φ) -> (√35/(4*√(2π))) * (sin(θ)^3)*(3*cos(φ)^2 - sin(φ)^2)*sin(φ) ) #fy(3x2-y2)
+        
+    push!(angular_parts, [-1,1] => (θ,φ) -> (1/√2)*s + (1/√2)*px ) #sp-1 
+    push!(angular_parts, [-1,2] => (θ,φ) -> (1/√2)*s - (1/√2)*px ) #sp-2
+    
+    push!(angular_parts, [-2,1] => (θ,φ) -> (1/√3)*s - (1/√6)*px + (1/√2)*py) #sp2-1
+    push!(angular_parts, [-2,2] => (θ,φ) -> (1/√3)*s - (1/√6)*px - (1/√2)*py) #sp2-2
+    push!(angular_parts, [-2,3] => (θ,φ) -> (1/√3)*s + (2/√6)*px )            #sp2-3
+    
+    push!(angular_parts, [-3,1] => (θ,φ) -> 0.5*(s(θ,φ)+px(θ,φ)+py(θ,φ)+pz(θ,φ)) )  #sp3-1
+    push!(angular_parts, [-3,2] => (θ,φ) -> 0.5*(s(θ,φ)+px(θ,φ)-py(θ,φ)-pz(θ,φ)) )  #sp3-2
+    push!(angular_parts, [-3,3] => (θ,φ) -> 0.5*(s(θ,φ)-px(θ,φ)+py(θ,φ)-pz(θ,φ)) )  #sp3-3
+    push!(angular_parts, [-3,4] => (θ,φ) -> 0.5*(s(θ,φ)-px(θ,φ)-py(θ,φ)+pz(θ,φ)) )  #sp3-4
+
+    push!(angular_parts, [-4,1] => (θ,φ) ->  (1/√3)*s - (1/√6)*px + (1/√2)*py)  #sp3d-1
+    push!(angular_parts, [-4,2] => (θ,φ) ->  (1/√3)*s - (1/√6)*px - (1/√2)*py)  #sp3d-2
+    push!(angular_parts, [-4,3] => (θ,φ) ->  (1/√3)*s + (2/√6)*px)              #sp3d-3
+    push!(angular_parts, [-4,4] => (θ,φ) ->  (1/√2)*pz + (1/√2)*dz2 )           #sp3d-4
+    push!(angular_parts, [-4,5] => (θ,φ) -> -(1/√2)*pz + (1/√2)*dz2 )           #sp3d-5
+
+
+    push!(angular_parts, [-5,1] => (θ,φ) -> (1/√6)*s - (1/√2)*px - (1/√12)*dz2 +  (1/2)*dx2_y2) #sp3d2-1
+    push!(angular_parts, [-5,2] => (θ,φ) -> (1/√6)*s + (1/√2)*px - (1/√12)*dz2 +  (1/2)*dx2_y2) #sp3d2-2
+    push!(angular_parts, [-5,3] => (θ,φ) -> (1/√6)*s - (1/√2)*py - (1/√12)*dz2 -  (1/2)*dx2_y2) #sp3d2-3
+    push!(angular_parts, [-5,4] => (θ,φ) -> (1/√6)*s + (1/√2)*py - (1/√12)*dz2 -  (1/2)*dx2_y2) #sp3d2-4
+    push!(angular_parts, [-5,5] => (θ,φ) -> (1/√6)*s - (1/√2)*pz + (1/√3)*dz2)                  #sp3d2-5
+    push!(angular_parts, [-5,6] => (θ,φ) -> (1/√6)*s + (1/√2)*pz + (1/√3)*dz2)                  #sp3d2-6
+    
+ 
+
+    radial_parts,angular_parts
+end
```

Comment:
Well I was hoping to avoid doing all this but if you've already done it... :D It's *very* easy to introduce bugs in tables like these (ask Michael, we had a fun bug in the spherical harmonics), which is why I'm always a bit suspicious...

Reply:
I haven't read the code very attentively but if you want to do them in real space, you have to periodize them, no? Eg if you have an atom at the corner of the cell it'll miss most of the function.

Reply:
Also this is a big code duplication with src/common/spherical_harmonics.jl. We can definitely refactor this and unify the implementations.